library/src/components-private/Error.vue

```diff
@@ -0,0 +1,86 @@
+<template lang="pug">
+  .container.pt-5.pb-10.error-page
+    .d-flex.justify-end
+      nio-button(caution-text @click="close") Close
+    .d-flex.justify-center
+      img.error-image(:src="imageUrl")
+    .heading.nio-h1.text-primary-darker.text-center.mt-5.mb-5 Oh no!
+    p.text-primary-dark.text-center.mb-0 There was an issue with this request. It's us, it's not you.
+    p.text-primary-dark.text-center.mt-0.mb-5 However please reach out to support and show them the error message below.
+    
+    NioTextArea.error-textarea.text-left.p-0(v-model="textError")
```

Comment:
I can still interact with the textarea and alter the text. Adding `pointer-events: none` in the styles seemed to do the trick.